src/storage.sh

```diff
@@ -44,9 +44,10 @@ __INTERNAL_STORAGE_DEFAULT_SECTION="GENERIC"
 __INTERNAL_STORAGE_DEFAULT_NAMESPACE="GENERIC"
 
 __INTERNAL_ST_OPTION_PARSER='
+  declare -r GETOPT_CMD="getopt"
   local namespace="$__INTERNAL_STORAGE_DEFAULT_NAMESPACE"
   local section="$__INTERNAL_STORAGE_DEFAULT_SECTION"
-  local GETOPT=$(getopt -o : -l namespace:,section: -- "$@" 2> >(while read -r line; do rlLogError "$FUNCNAME: $line"; done)) || return 126
+  local GETOPT=$($GETOPT_CMD -o : -l namespace:,section: -- "$@" 2> >(while read -r line; do rlLogError "$FUNCNAME: $line"; done)) || return 126
```

Comment:
For the most part I think it will work fine, this is the only part that I have my doubts

Reply:
since the `__INTERNAL_STORAGE_DEFAULT_NAMESPACE` are used with the definition from outside the scope, I think this will be fine